matrix-sdk-android/src/main/java/org/matrix/android/sdk/api/MatrixConfiguration.kt

```diff
@@ -61,14 +61,14 @@ data class MatrixConfiguration(
          * RoomDisplayNameFallbackProvider to provide default room display name.
          */
         val roomDisplayNameFallbackProvider: RoomDisplayNameFallbackProvider,
-        /**
-         * True to enable presence information sync (if available). False to disable regardless of server setting.
-         */
-        val presenceSyncEnabled: Boolean = true,
         /**
          * Thread messages default enable/disabled value
          */
         val threadMessagesEnabledDefault: Boolean = false,
+        /**
+         * Presence offline mode default enabled/disabled value
+         */
+        val presenceOfflineModeEnabledDefault: Boolean = false,
```

Comment:
I think naming could maybe be a bit clearer.
What about: `userAlwaysAppearsOfflineByDefault`?

Other names could be updated using the same terminology.

Reply:
(But I think this should be removed anyway)